core/services/job/orm.go

```diff
@@ -119,6 +119,7 @@ func (o *orm) ClaimUnclaimedJobs(ctx context.Context) ([]SpecDB, error) {
 	err := o.db.
 		Joins(join, args...).
 		Preload("OffchainreportingOracleSpec").
+		Preload("PipelineSpec.PipelineTaskSpecs").
```

Comment:
we save a read by just populating everything in the job spec here

Reply:
Can't see where this is used?

Reply:
its used instead of reading again here https://github.com/smartcontractkit/chainlink/pull/3954/files#diff-8c4ea9636fec98879a197b828c064da206d788de90a0c6754b413dbe6f0891cbL28

Reply:
But thats deleted...